src/traces/contour/plot.js

```diff
@@ -636,8 +636,6 @@ function clipGaps(plotGroup, plotinfo, clips, cd0, perimeter) {
     else clipId = null;
 
     plotGroup.call(Drawing.setClipUrl, clipId);
-    plotinfo.plot.selectAll('.hm' + cd0.trace.uid)
-        .call(Drawing.setClipUrl, clipId);
```

Comment:
Alright, GH is working again with these images, must've been an internal glitch last night.

The reason for the image diffs is that in master (right) we were applying the `clip-path` to both the heatmap group and the contour group... and now that the heatmap is inside the contour group that was redundant but apparently it was still having an effect on antialiasing. The `Drawing.setClipUrl ` here that was supposed to be clipping the heatmap actually wasn't doing anything, since I removed the uid-based class from the heatmap.
<img width="1155" alt="screen shot 2018-08-17 at 9 59 51 am" src="https://user-images.githubusercontent.com/2678795/44270786-6ef73b80-a206-11e8-9ba4-dac9086d16a1.png">
So in fact the image diff in this PR probably nearly (or exactly) reverts the diff from `contour-legend` https://github.com/plotly/plotly.js/pull/2891/files?short_path=2b47456#diff-2b474566a55f847908961b61d6e0cba4